### PR TITLE
CS-558-2 Location widget updates

### PR DESCRIPTION
**Pull request by jookyg**

### CODE REVIEW

File: web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js [hunk 1]
```diff
@@ -791,31 +945,207 @@
...
+              location_type = removeTrailingComma(location_type);
+              internal_details = removeTrailingComma(internal_details);
+
+              //$('input[name=' + elementId + '\\[place_name\\]]').val(place_name);
+              $('input[name=' + elementId + '\\[location_type_hidden\\]]').val(location_type);
+              $('input[name=' + elementId + '\\[location_attributes\\]]').val(internal_details);
+
+              // set location radio button
+              // report_location[location_type]
+              var radioInputs = $('input[name="report_location[location_type]"]');
```
Comment (odensc) [inline web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js:1068]: This should probably use `elementId`.

**Review by odensc — CHANGES_REQUESTED** [review]

File: web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js (hunk 1 above)
Comment (jookyg) [reply web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js]: Fixed. Good catch!

Comment (jookyg) [reply web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js]: This is strange. I've updaed and pushed the file to fix this issue, but the PR is still showing the old version. Though I can see the commit with the change in the log.

Comment (jookyg) [reply web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js]: @odensc can you see this as fixed on line 1068?

Comment (odensc) [reply web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js]: @jookyg Yup, it shows up for me. Super weird! I guess it's fine if I can see it 😁

**Pull request merged**